tests/HEDA.robot

```diff
@@ -0,0 +1,266 @@
+*** Settings ***
+
+Resource       cumulusci/robotframework/Salesforce.robot
+Library        tests/HEDA.py
+
+*** Keywords ***
+
+
+API Create Term
+    [Arguments]       ${account_id}  &{fields}
+    ${term_name} =    Generate Random String
+    ${term_id} =      Salesforce Insert  Term__c
+    ...                   Name=${term_name}
+    ...                   Account__c=${account_id}
```

Comment:
You need to add a line after the Account__c line like below to actually pass in the custom field values:
`...                   Account__c=${account_id}`
`...                   &{fields}`